README.md

```diff
@@ -1,7 +1,7 @@
 #Pome
 Pome stands for **Po**stgres **Me**trics.
 Pome is a PostgreSQL Metrics Dashboard to keep track of the health of your database.
-This project is at a very early stage and there are a lot of missing features,
+This project is at a very early stage and there are a lot of missing features
```

Comment:
@rach @rlayte Is there anything else I need to do to improve this PR? Since we arrived at a compromise, about the contentious comma placement issue, I thought the rest of the corrections were okay, no?